KIPs/kip-7-fungible_token.md

```diff
@@ -170,7 +211,39 @@ OPTIONAL - This method can be used to improve usability,
 but interfaces and other contracts MUST NOT expect these values to be present.
 
 ```solidity
-function mint(address _to, uint256 _value) public returns (bool) {
+function mint(address _to, uint256 _value) public onlyMinter returns (bool) {
```

Comment:
Do we need to have function modifier (e.g., onlyMinter)?

Is there any description of onlyMinter?

Reply:
Actually, I didn't describe the modifier 1st time.
But this time, I just copied the function from openzeppline. 
Because, I thought removing the modifier from reference code is weird.
As you know, modifier doesn't effect to the hash of the function. 

What if we explain that the modifiers can be different?

Reply:
First, the modifier logic should be described in the function somewhere.
(e.g., mint() MUST `throw` if the sender is not a minter.)

Second, we do not have to enforce unnecessary parts. We can describe the specification of mint() without onlyMinter.

Reply:
I got it. I will remove modifiers. :)